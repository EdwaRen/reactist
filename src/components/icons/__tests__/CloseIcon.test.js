import React from 'react'
import { shallow } from 'enzyme'

import CloseIcon from '../CloseIcon.svg'

describe('CloseIcon', () => {
    it('renders a svg icon', () => {
        const closeIcon = shallow(<CloseIcon />)
        expect(closeIcon).toMatchSnapshot()
    })
})
