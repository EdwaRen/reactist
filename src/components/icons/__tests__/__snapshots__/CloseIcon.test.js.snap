// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon renders a svg icon 1`] = `
<svg
  height="12"
  viewBox="0 0 12 12"
  width="12"
>
  <path
    d="M12 1.427L10.573 0 5.997 4.576 1.424.003l-1.42 1.42 4.572 4.574L0 10.573 1.427 12l4.576-4.576 4.573 4.573 1.42-1.42-4.572-4.574L12 1.427z"
    fill="#B3B3B3"
    fillRule="evenodd"
  />
</svg>
`;
